### PR TITLE
feat: jumbotron

### PR DESCRIPTION
**Pull request by assasin0076**

### CODE REVIEW

**Comment by assasin0076** [comment]
https://github.com/themesberg/flowbite-vue/issues/210

File: docs/components/jumbotron.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+
+
+## Default Jumbotron
+Use this default example to show a title, description, and two CTA buttons for the jumbotron component.
+
+<fwb-jumbotron-example />
+```vue
+<template>
+  <div
+    class="vp-raw"
```
Comment (cogor) [inline docs/components/jumbotron.md:27]:
```suggestion

```

File: docs/components/jumbotron.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+```
+
+## Background image
+Use this jumbotron to apply a background image with a darkening opacity effect to improve readability.
+
+<fwb-jumbotron-background-image-example />
+```vue
+<template>
+  <div
+    class="vp-raw"
```
Comment (cogor) [inline docs/components/jumbotron.md:79]:
```suggestion

```

File: docs/components/jumbotron.md [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+```
+
+## Featured video
+This component can be used to feature a video together with a heading title, description, and CTA buttons.
+
+<fwb-jumbotron-video-example />
+```vue
+<template>
+  <div
+    class="vp-raw"
```
Comment (cogor) [inline docs/components/jumbotron.md:134]:
```suggestion

```

File: docs/components/jumbotron.md [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+```
+
+## Authentication form
+Use this component to show a sign in or register form as the first section of your website.
+
+<fwb-jumbotron-form-example />
+```vue
+<template>
+  <div
+    class="vp-raw"
```
Comment (cogor) [inline docs/components/jumbotron.md:199]:
```suggestion

```

**Pull request merged**